java/grpc/conf/application.conf

```diff
@@ -56,17 +54,8 @@ pekko.grpc.client {
     host = ${?DEPLOYMENT_SERVICE_NAME}
     port = ${play.server.https.port}
     use-tls = true
-    # The Authority on the requests must match the CN on the server certificate. The certificate on
+    # The Authority on the requests must match the CN on the server certificate. The self-signed certificate certificate on
     # `conf/selfsigned.keystore` was issued to `localhost` so we must override the authority.
     override-authority = "localhost"
-
-    # configure ssl to trust our fake certificate chain.
-    ssl-config {
-      trustManager = {
-        stores = [
-          {type = "JKS", path = ${user.dir}/conf/selfsigned.keystore}
-        ]
-      }
-    }
```

Comment:
pekko/akka-grpc does not have a ssl-config settings key anymore
Since
https://github.com/apache/pekko-grpc/commit/1c7e655ac9a495468cfdfe486f9270db0aac9515
(see this commit's reference.conf)

Also see current reference.conf:
https://github.com/apache/pekko-grpc/blob/v1.1.1/runtime/src/main/resources/reference.conf